src/partials/ImageInput.svelte

```diff
@@ -45,13 +46,12 @@
               body.append("file[]", file)
             }
 
-            const result = await uploadToNostrBuild(body)
-
-            // Legacy weirdness
-            for (const {url} of result.data) {
-              value = url
-              onChange?.(url)
+            const result = await uploadToMediaProvider(settings.mediaprovider_url, body)
+            if (result != "" && result != null){
```

Comment:
type coercion is fine, `if (result) {`. Nit aside, it looks like this removes the ability to upload multiple images simultaneously.

Reply:
I'm going to see if I can do something with this

Reply:
done